apps/api-documenter/src/markdown/test/__snapshots__/CustomMarkdownEmitter.test.ts.snap

```diff
@@ -50,9 +50,34 @@ HTML escape: &amp;quot;
 
 ## Table
 
-|  Header 1 | Header 2 |
-|  --- | --- |
-|  Cell 1 | <p>Cell 2</p><p>**bold text**</p> |
+<table><thead><tr><th>
 
+Header 1
+
+
+
+</th><th>
+
+Header 2
+
+
+
+</th></tr></thead>
+<tbody><tr><td>
+
+Cell 1
+
+
+
```

Comment:
Can we emit two newlines after the cell contents instead of four?

Reply:
I believe only two of them are from this PR, and the other two might come from `writeNode`.

I have to admit I don't know a lot about the `writeNode` implementation and if it always does that, or only in special cases.

I guess I went a bit overboard here and would be open for changes.

Maybe something like

```diff
-        writer.writeLine();
+        writer.ensureNewLine();
        writer.writeLine();
```

in the places where I applied two `writeLine`s?

Generally: I'm already in bed at this point, so I can't do any more changes today, but I would honestly be over the moon if I wake up to see this merged tomorrow (I essentially have a package release that's just waiting for a docs PR that relies on this 😅) - so if that's in any way a realistic expectation, please just go ahead and edit whatever you want here :)

Reply:
Okay, we're down to three newlines, two of which come from the `writeNode` call  - is that acceptable?